spec/truffle/interop/foreign_object_spec.rb

```diff
@@ -23,4 +23,10 @@
     }.should output_to_fd("#{foreign.inspect}\n")
   end
 
+  it "handles as falsy value" do
+    js_null = Truffle::Debug.foreign_null ? true : false
```

Comment:
`js_null` => `foreign_null`

Also this should be in `spec/truffle/interop/coercion_spec.rb`, and include `null` in the description (foreign objects in general are not false, only foreign nulls).